config/_config.mixins.scss

```diff
@@ -49,3 +49,30 @@ Ideally, all mixins should be declared here but in some cases, they've been decl
     -moz-appearance: $value;
     appearance: $value;
 }
+
+/* clearfix
```

Comment:
Why did we need this done as part of this branch?

I'm thinking this be ported into another branch todo for the future.

Reply:
ok have transferred this to another branch for now